docs/spark-mlcontext-programming-guide.md

```diff
@@ -284,6 +334,30 @@ df: org.apache.spark.sql.DataFrame = [C0: double, C1: double, C2: double, C3: do
 {% endhighlight %}
 </div>
 
+<div data-lang="Python" markdown="1">
+{% highlight python %}
+numRows = 10000
+numCols = 100
+from random import random
+from pyspark.sql.types import *
+data = sc.parallelize(range(numRows)).map(lambda x : [ random() for i in range(numCols) ])
+schema = StructType([ StructField("C" + str(i), DoubleType(), True) for i in range(numCols) ])
+df = spark.createDataFrame(data, schema)
```

Comment:
Side note: I realize this example is following the existing Scala one, but at some point it would be cleaner to update both of these to use `Vector` examples.  Otherwise, this appears to show that SystemML requires more complexity than necessary.

```
num_rows = 10000
num_cols = 10
from random import random
data = sc.parallelize(range(num_rows)).map(lambda x : ([random() for i in range(num_cols)],))
df = data.toDF(["features"])
```